lib/elixir/lib/enum.ex

```diff
@@ -2484,8 +2484,8 @@ defmodule Enum do
 
   ## shuffle
 
-  defp unwrap([{_, h} | collection], t) do
-    unwrap(collection, [h|t])
```

Comment:
this is the only change we should keep. all other ones are just changing syntax and should be avoided because there is no style guide. your code is as correct or as wrong as the one before.